### PR TITLE
AAD SOO through opdg not supported in vnets

### PR DESCRIPTION
**Pull request by nikkiwaghani**

### CODE REVIEW

**Comment by jborsecnik** [comment]
@KesemSharabi, 

Can you review the proposed changes? 

IMPORTANT: When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Comment by PRMerger16** [comment]
@nikkiwaghani : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger7** [comment]
@nikkiwaghani : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by KesemSharabi** [comment]
@arthiriyer, can you please review the proposed change?

File: powerbi-docs/connect-data/service-gateway-azure-active-directory-sso.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ LocalizationGroup: Gateways
 
 Azure AD SSO enables single sign on on the data gateway to cloud data sources that rely on Azure Active Directory (Azure AD)-based authentication. When you configure Azure AD SSO on the on-premises data gateway for an applicable data source, queries execute under the Azure AD identity of the user that interacts with the Power BI report.
 
-While Azure Virtual Networks (VNets) offer network isolation and security for your resources on the Microsoft cloud, you now require a secure way to connect to these data sources. On-premises data gateways help you achieve that. Additionally, Azure AD SSO as explained above allows users to see only data that they have access to.
+While Azure Virtual Networks (VNets) offer network isolation and security for your resources on the Microsoft cloud, you now require a secure way to connect to these data sources. On-premises data gateways help you achieve that. Azure AD SSO is not currently supported through the virtual network data gateway solution.
```
Comment (nikkiwaghani) [inline powerbi-docs/connect-data/service-gateway-azure-active-directory-sso.md:18]: #sign-off

**Comment by PRMerger-2** [comment]
Invalid command: '#sign-off'. Only the assigned author of one or more file in this PR can sign off. @arthiriyer

**Pull request closed without merging**